packages/playground/src/App.js

```diff
@@ -260,31 +260,32 @@ export default class extends React.Component {
                 value={ state.language }
               >
                 <option value="">Default ({ window.navigator.language })</option>
+                <option value="zh-HK">Chinese (Hong Kong)</option>
```

Comment:
Re-ordering. For UX reason, we re-order based on the text, instead of the value, which is a little bit painful, but better for the user.